all_msgs/ros2interop/ros-types-all.idl

```diff
@@ -0,0 +1,1825 @@
+module std_msgs {
+  module msg {
+    module dds_ {
+      @appendable
+      struct MultiArrayDimension_ {
+                string<255> label;
```

Comment:
Use 4 spaces here.

Reply:
This file was the unmodified output of rtiddsgen. 
Will modify spaces in structs, and before 'typedefs' to make consistent.
Should this also include a copyright comment header in the file?